decoder/vaapidecoder_h264.cpp

```diff
@@ -1760,7 +1760,7 @@ YamiStatus VaapiDecoderH264::decodeSlice(NalUnit* nalu)
         status = createPicture(slice, nalu);
         if (status != YAMI_SUCCESS)
             return status;
-        if (!m_currPic
+        if (!m_currPic || !m_prevPic
```

Comment:
This mean after seek the first frame is not IDR? If it is an IDRit should set by https://github.com/01org/libyami/blob/apache/decoder/vaapidecoder_h264.cpp#L1721